ignite/contrib/handlers/clearml_logger.py

```diff
@@ -178,6 +180,20 @@ def bypass_mode(cls) -> bool:
         """
         return getattr(cls, "_bypass", bool(os.environ.get("CI")))
 
+    def get_clearml_task(self) -> Union[None, Task]:
+        """
+        Returns the ClearML Task.
+
+        Return:
+            If bypass mode state is ``True``, all outside communication is skipped,
+            and ``get_task`` will return ``None``.
+            If bypass mode state is ``False``, this method will return the ClearML Task.
+        """
+        if self.bypass_mode():
+            return None
+
+        return self._task
```

Comment:
```suggestion
        return self._task
```
Let's just return `self._task`